app/views/network_router/remove_interface_select.html.haml

```diff
@@ -9,13 +12,12 @@
       .col-md-8
         = select_tag("cloud_subnet_id",
           options_for_select([["<#{_('Choose')}>", nil]] + @subnet_choices.sort),
-                      "ng-model"                    => "networkRouterModel.cloud_subnet_id",
+                      "ng-model"                    => "vm.networkRouterModel.cloud_subnet_id",
                       "required"                    => "",
-                      :checkchange                  => true,
                       "selectpicker-for-select-tag" => "")
-  = render :partial => "layouts/angular/x_custom_form_buttons_angular",
+  = render :partial => "layouts/angular/custom_form_buttons_controller_as",
```

Comment:
Can you add a comment to `x_custom_form_buttons_angular` to indicate that the controllerAs replacement is called `custom_form_buttons_controller_as` please? :)